Tests/RiskTests/RiskTests.swift

```diff
@@ -10,38 +10,141 @@ import XCTest
 @testable import Risk
 
 class RiskTests: XCTestCase {
-
-    // TODO: Update CI to pass environment variable
-    // func testGetInstanceWithValidConfiguration() {
-    //     let expectation = self.expectation(description: "Risk instance creation")
-    //     var createdRiskInstance: Risk?
-        
-    //     guard let publicKey = ProcessInfo.processInfo.environment["SAMPLE_MERCHANT_PUBLIC_KEY"] else {
-    //         XCTFail("Environment variable SAMPLE_MERCHANT_PUBLIC_KEY is not set.")
-    //         return
-    //     }
-
-    //     let validConfig = RiskConfig(publicKey: publicKey, environment: RiskEnvironment.qa)
-
-    //     Risk.getInstance(config: validConfig) { risk in
-    //         createdRiskInstance = risk
-    //         expectation.fulfill()
-    //     }
-
-    //     waitForExpectations(timeout: 5, handler: nil)
-    //     XCTAssertNotNil(createdRiskInstance)
-    // }
-
+    
+//    func testGetInstanceWithValidConfiguration() {
```

Comment:
also this wouldn't help anymore. We can keep this commented out code somewhere else internally, if we think we'll need it